lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -372,11 +377,12 @@ defmodule ExUnit.Runner do
   defp process_max_failures(%{max_failures: :infinity}, _), do: :no
 
   defp process_max_failures(config, %ExUnit.TestModule{state: {tag, _}, tests: tests})
-       when tag in [:failed, :invalid] do
+       when tag in [:failed] do
```

Comment:
Let's pattern match on `:failed` then instead of the guard clause. :)

Reply:
fixed